templates/pke/master.cf.yaml

```
@@ -20,6 +20,10 @@ Parameters:
   VPCId:
     Type: 'AWS::EC2::VPC::Id'
     Description: Specify VPC Id for Autoscaling
+  VPCDefaultSecurityGroupId:
+    Type: String
+    Description: Default security group of the VPC
+    Default: ""
```

should not have a default (it will fail later if not specified, won't it?)